src/dev/src/app/tabs/tabs-static.demo.html

```diff
@@ -3,45 +3,94 @@
       </li>
-      <div class="dropdown-menu">
-        <button class="btn" aria-controls="panelInfrastructure">Infrastructure</button>
-        <button class="btn" aria-controls="panelSettings">Settings</button>
+      <li role="tab" class="nav-item">
+        <button id="tabManagement" class="btn btn-link nav-link" aria-controls="panelManagement" aria-selected="false" type="button">Management</button>
+      </li>
+      <li role="tab" class="nav-item">
+        <button id="tabCloud" class="btn btn-link nav-link" aria-controls="panelCloud" aria-selected="false" type="button">Cloud</button>
+      </li>
+      <div class="tabs-overflow open bottom-right">
+        <li class="nav-item">
+          <button class="btn btn-link nav-link dropdown-toggle">
+            <clr-icon shape="ellipsis-horizontal" class="is-info" title="More"></clr-icon>
+          </button>
+        </li>
+        <div class="dropdown-menu">
+          <button class="btn" role="tab" aria-controls="panelInfrastructure">Infrastructure</button>
+          <button class="btn" role="tab" aria-controls="panelSettings">Settings</button>
+        </div>
       </div>
-    </div>
-  </ul>
-  <section id="panelDashboard" role="tabpanel" aria-labelledby="tabDashboard">
-    <p>Content for Dashboard</p>
-  </section>
-  <section id="panelManagement" role="tabpanel" aria-labelledby="tabManagement" aria-hidden="true">
-    <p>Content for Management</p>
-  </section>
-  <section id="panelCloud" role="tabpanel" aria-labelledby="tabCloud" aria-hidden="true">
-    <p>Content for Cloud</p>
-  </section>
-  <section id="panelInfrastructure" role="tabpanel" aria-labelledby="tabInfrastructure" aria-hidden="true">
-    <p>Content for Infrastructure</p>
-  </section>
-  <section id="panelSettings" role="tabpanel" aria-labelledby="tabSettings" aria-hidden="true">
-    <p>Content for Settings</p>
-  </section>
+    </ul>
+    <section id="panelDashboard" role="tabpanel" aria-labelledby="tabDashboard">
+      <p>Content for Dashboard</p>
+    </section>
+    <section id="panelManagement" role="tabpanel" aria-labelledby="tabManagement" aria-hidden="true">
+      <p>Content for Management</p>
+    </section>
+    <section id="panelCloud" role="tabpanel" aria-labelledby="tabCloud" aria-hidden="true">
+      <p>Content for Cloud</p>
+    </section>
+    <section id="panelInfrastructure" role="tabpanel" aria-labelledby="tabInfrastructure" aria-hidden="true">
+      <p>Content for Infrastructure</p>
+    </section>
+    <section id="panelSettings" role="tabpanel" aria-labelledby="tabSettings" aria-hidden="true">
+      <p>Content for Settings</p>
+    </section>
+  </div>
+
+  <h3>Vertical tabs</h3>
+  <div class="demo tabs-vertical">
+    <ul id="verticalDemoTabs" class="nav" role="tablist">
+      <li role="tab" class="nav-item">
+        <button id="verticalTabDashboard" class="btn btn-link nav-link active" aria-controls="panelDashboard" aria-selected="true" type="button">Dashboard</button>
```

Comment:
Do we need the `id` attribute on all these buttons? People tend to copy/paste from our demos so would like them to be clean.

Reply:
I think I intended to use them in some gemini tests about hovering vs selected style, but I then simplified them and now the ids may be useless. Will check it and update here soon.
Though this is the dev-app, not the website and whoever can get to here should be advanced enough to be able to copy smartly.

Reply:
No, it's not me, I've just conformed to the style of the other elements we already had on this page. Nevertheless, these IDs are not used anywhere, so I've removed them altogether.
Fixed.